### PR TITLE
Issues/165

### PR DESCRIPTION
**Pull request by wmwv**
Example notebook for loading a merged tract+patch from an HDF5 file and checking the stellar color-color locus.

### CODE REVIEW

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+    "plt.ylabel('r-i')\n",
+    "\n",
+    "plot_stellar_locus('gmr', 'rmi')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Clean this up a bit to only include stars with SNR > 3 in g and r"
```
Comment (djperrefort) [inline Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: You say you are going to implement a cut for SNR > 3, but then impliment 3 different cutoffs. This left me with some minor but unnecessary confusion. Explain first that you are going to use different SNR to create 3 different catagories: good snr, bright stars, and galaxies.

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+    "## Generic Catalog Reader (GCR)\n",
+    "\n",
+    "After a little bit you'll find yourself wanting to ask interesting questions, like how well does the DC2 analysis recover the input sources?  Or easily loading in all of the patches of a tract.  A convenient framework for generalizing analyses of DESC-DC related catalogs is the Generic Catalog Reader:\n",
+    "\n",
+    "* GCR is a general utility\n",
+    "https://github.com/yymao/generic-catalog-reader\n",
+    "* The catalogs supporting DC2 are in:\n",
+    "https://github.com/LSSTDESC/gcr-catalogs\n",
+    "\n",
+    "See 'DC2_Static_Coadd_Catalog_reader.ipynb' notebook by Yao-Yuan Mao\n",
```
Comment (djperrefort) [inline Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: Link to DC2_Static_Coadd_Catalog_reader.ipynb for easy navigation.

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb [hunk 3]
```diff
@@ -0,0 +1,503 @@
...
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(8,6.5))\n",
+    "plt.hist2d(df['g_mag']-df['r_mag'], df['r_mag']-df['i_mag'],\n",
+    "           cmap=cmap, range=((-1, +2), (-1, +2)), bins=51)\n",
+    "plt.colorbar()\n",
+    "plt.xlabel('g-r')\n",
+    "plt.ylabel('r-i')\n",
+    "\n",
+    "plot_stellar_locus('gmr', 'rmi')"
```
[the remark this answers is not in the document]
Comment (wmwv) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: From what directory are you running the notebook?

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb (hunk 2 above)
Comment (wmwv) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: The next line is that link.  Does it show up for you?

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (djperrefort) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: Just my user directory. I did a little directory tree exploration for the file but didn't see it.

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb (hunk 2 above)
Comment (djperrefort) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: Ah yes, for some reason I had thought that was something else. Not the notebook's fault, just a personal mental skip.

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (wmwv) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: It's in the same directory as the notebook.  Did you use the NERSC Jupyter-dev interface to run this?

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb [hunk 4]
```diff
@@ -0,0 +1,503 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "def simple_stellar_locus_gmr_rmi(color='red', linestyle='--', linewidth=2.5):\n",
```
[the remark this answers is not in the document]
Comment (wmwv) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: 👍

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb [hunk 5]
```diff
@@ -0,0 +1,503 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "z = galaxies\n",
+    "plt.scatter(z['g_mag'] - z['r_mag'], z['g_mag'], marker='.')\n",
```
[the remark this answers is not in the document]
Comment (wmwv) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: +1

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb (hunk 1 above)
Comment (wmwv) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: 👍

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (djperrefort) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: When ssh-ing into cori and cloning the DC2_Repo repo this error does not occur. Previously had an issue with my ssh access and was using the Jupyter upload interface. I simply forgot to upload this file.

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb [hunk 6]
```diff
@@ -0,0 +1,503 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nbpresent": {
+     "id": "0789d321-e15e-44b8-b96e-34b4e993cf16"
+    }
+   },
+   "source": [
+    "# Looking at merged tract-patch catalogs in Run 1.1 leading up to DC2\n",
```
[the remark this answers is not in the document]
Comment (wmwv) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: 👍

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb (hunk 2 above)
Comment (wmwv) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: 👍

File: Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (wmwv) [reply Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb]: 👍

**Comment by wmwv** [comment]
@yymao Agreed.  I hope that we can stop at four.  1) GCR 2) DB 3) Merged catalog 4) Butler-based

**Pull request merged**